reactive/zookeeper.py

```diff
@@ -58,4 +55,5 @@ def quorum_remove(zkpeer):
 @when('zookeeper.installed', 'zkclient.joined')
 def serve_client(client):
     port = DistConfig().port('zookeeper')
-    client.send_port(port)
+    rest_port = DistConfig().port('zookeeper-rest')
```

Comment:
My Python aesthetic says don't parse the yaml twice, so we should save a copy of the `DistConfig()` instance and use it.